### DISCUSSION TITLE
2022-11-02 Weekly Meeting 80 Agenda

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#310] [Weekly Meetings]
Weekly Meeting 80 is happening on Wednesday November 2, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [UX Research](https://github.com/omigroup/ux-research) 
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [OMI Website update](https://github.com/omigroup/website)
- any others?

### Discussions: 
- https://github.com/omigroup/website/issues/57 - Lets explore whats available in wordpress and what might fit in in conjunction with using github.
#### Next week: 
- https://github.com/omigroup/omigroup/issues/234 - Audit existing bots
- https://github.com/omigroup/omigroup/issues/289 - Proposed migrations: Jitsi* and ProtonMail/Cal and Element/Matrix

### Future demo themes: 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
Nice script!

**Comment by mrmetaverse** [comment]
(from backlog refinement 11/1/22) https://github.com/omigroup/omigroup/issues/289

**Reply by mrmetaverse** [reply]
https://github.com/omigroup/website/issues/57

**Reply by mrmetaverse** [reply]
#234

**Comment by mrmetaverse** [comment]
# Opening chat 

We discussed different business models and hybrid open-source approaches to features like massive-multiplayer as a service, or portals as a service.  

Some will operate under the assumption that builders are willing to build inside of their proprietary system in exchange for something like multiplayer capabilities. Others will build as-a-service approaches, where you can keep your own servers, but plug-in to their multiplayer or inventory/wearables, or portals as a service. 

We gave little shoutouts to framevr and cryptovoxels for not requiring users to create an account to access the experiences: https://framevr.io/fearless-xrhq

**Comment by mrmetaverse** [comment]
# Documentation Upgrades 

- humbletim set up a script to send agendas to github, this may provide a huge value add to the working groups as well. 
- antpb: there are features in Wordpress like to enable a certain interval of time to run a certain action, and the ability to store the API keys in a secure way to generate agendas from the website to github.

**Comment by mrmetaverse** [comment]
# Backlog grooming 

- setting ourselves up for a new wave of project management 
- https://github.com/omigroup/omigroup/discussions/311
- archived dead tix, these can still be found and reopened as needed.

**Comment by mrmetaverse** [comment]
# MSF-Delegates 

Meeting cadences are starting to shift as working groups in MSF properly form. These are still very much the early days, with the standard growing pains like "too many channels," etc. We make sense of these in our MSF-Opportunities channel in the OMI / Angell Discord. 

If you have ideas for ways we can improve how we share out this information please let us know as well. 
![image](https://user-images.githubusercontent.com/63426722/199544860-e6243325-08ce-43e7-ba92-629adaaa929b.png)

**Comment by mrmetaverse** [comment]
# UX  Research Group 
- 2nd meeting was yesterday
- https://github.com/omigroup/ux-research/discussions/13 (click for links to the referenced items)
- exploring tools like frame and spatial.io for "war rooms" like @madjin was talking about in previous weeks. 
- ![image](https://user-images.githubusercontent.com/63426722/199545289-4587dd7f-eb58-4ebf-a508-123f21ac481d.png)
- Explored suggestions made on https://github.com/omigroup/ux-research/issues/12 in a new figjam space
- ![image](https://user-images.githubusercontent.com/63426722/199545445-0ea754f8-c896-40c6-b09e-b5fcb8be364a.png)
- **Exploring ways to have more events and meetings in virtual spaces, or hybrid-virtual spaces.** 
- Next meeting is on November 15th at 16:00 UTC! 
- Evo has extensive experience designing for synchronization and collaboration, like for virtual drum circles / yoga sessions, which we can likely learn from.

**Comment by mrmetaverse** [comment]
# Scripting
[Scripting group](https://github.com/omigroup/omi-scripting-group)

**Reply by mrmetaverse** [reply]
No shared updates

**Comment by mrmetaverse** [comment]
# OMI developer hour

Next meetup is tomorrow November 3, 2022 at 17:00 UTC
Maybe we can try out a different developer hour time, as a like a Friday creator hour session. 
Evo is open to streaming it. Some time between 1-5pm PT (17:00 UTC-ish)

**Comment by mrmetaverse** [comment]
# gLTF extensions 

- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
- Progress on OMI_ref and authoring extensions
- The group discussed OMI collider and Audio Emitter. 
  - Emitter has a small bug that should be resolved in the next week :) 

**New:** The working group discussed a new OMI-Link extension. The [drafted proposal by @antpb](https://github.com/omigroup/gltf-extensions/pull/115). This is very much what a lot of people would refer to as "portals". The proposal came from some common ground identified between experiments with both Wordpress and Matrix based solutions. There may be other metadata we want to consider. Ant suggested exploring anchor tags as a reference.

**Reply by mrmetaverse** [reply]
SOLID example of how we can take our own private work and use it as inspo for suggested extensions.

**Comment by mrmetaverse** [comment]
# Media 
[Media Group](https://github.com/omigroup/media)

- Group to meet to discuss changing twitter strategy. We have a mastodon account which we may want to lean in on. 
- Group to continue discussing community brand guidelines in next Media meeting. 
- Creating a speakers form for those who are interested in speaking at events as part of OMI. 
- If you would like to participate in a panel together, that is something we can put together as soon as this month. Evo can put together a panel that discusses OMI, what we do, or other interesting topics that may be good for an event next year, let's talk about putting some panels together. 
- Next meeting is next Tuesday November 8, 2022 at 15:30 UTC

**Reply by mrmetaverse** [reply]
event details ad-hoc living in the media group. We may branch off into proposing a working group if needed in the future.

**Comment by mrmetaverse** [comment]
# M3 Avatar Interop 
[M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)

- Jin was exploring a hyperfi space, and an openvoxels project with cryptovoxels wearables scattered across a desert. It was really cool!

**Comment by mrmetaverse** [comment]
# Website
[OMI Website update](https://github.com/omigroup/website)

- https://github.com/omigroup/website/issues/57
- maybe leveraging wordpress for gathering form data, etc. 
- maybe a job-board, taking the discord channel to the next level (`metaverse-jobs-board`) 
- guest blog writing access 
  - editorial flow where contributors can publish content which goes to the editor to publish. 
  - a number of built-in features for this. 
- We can schedule a demo by chatting in discord. Ideally appx 5pm Central time (22:00 UTC)

**Reply by mrmetaverse** [reply]
- maybe leveraging wordpress for gathering form data, etc. 
- maybe a job-board, taking the discord channel to the next level (`metaverse-jobs-board`) 
- guest blog writing access 
  - editorial flow where contributors can publish content which goes to the editor to publish. 
  - a number of built-in features for this. 
- We can schedule a demo by chatting in discord. Ideally appx 5pm Central time (22:00 UTC)
